services/src/main/java/org/fao/geonet/services/thesaurus/GetTopConcept.java

```diff
@@ -74,6 +74,7 @@ public Element exec(Element params, ServiceContext context)
 
             KeywordBean topConcept = new KeywordBean(the.getIsoLanguageMapper());
             topConcept.setThesaurusInfo(the);
+            topConcept.setValue("topConcepts", "eng");
```

Comment:
Doesn't this change seem wrong, isn't it overridden by the next line?

Reply:
Not really, if you're using the UI in another language. Some UI checks could be wrong in some situation eg. `data-ng-disabled="concept.prefLabel[language] == 'topConcepts'"`

<img width="617" height="117" alt="Image" src="https://github.com/user-attachments/assets/33420545-492b-4c49-b4ef-7a35cbb71f5a" />


So here it ensure the prefLabel of the (not displayed) top concept has an english label that we can check.